### PR TITLE
Feature sass

### PR DESCRIPTION
**Pull request by H-Bo214**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features

### Detailed Description

- Update/add alt text to main html page, add some variables to sass file for frequently used colors and color/hover to the buttons within the recipe
- Update to filter section position
- Add function searchSavedRecipes
- Add mobile layout, implement sass extend
- Remove some white space
- Separated some sass files 

### Files modified:
- [ ] domUpdates.js
- [ ] styles.scss
- [ ] _header.scss
- [ ] _nav.scss
- [ ] _variables.scss
- [ ] scripts.js
- [ ] Other files:

### CODE REVIEW

File: src/css/_nav.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+.nav-btn:hover {
+    color: $primary-green;
+    font-size: 14px;
+    text-shadow: 0px 2px 4px black;
+    transition: .4s ease;
+}
+
+.nav-btn img {
```
Comment (lilianaweimer) [inline src/css/_nav.scss:26]: We may be able to nest these (`.nav-btn:hover` and `.nav-btn img`) inside `.nav-btn`

File: src/css/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+$primary-green: #C4EB67;
+$secondary-green: #359567;
+$main-and-button-font: "Quicksand", sans-serif;
+$h1-font: "Sulphur Point", sans-serif;
```
Comment (lilianaweimer) [inline src/css/_variables.scss:4]: This has a thing that says "no newline at end of file" next to it, any idea what that means?

File: src/css/styles.scss [hunk 3]
```diff
@@ -381,3 +337,53 @@ main {
...
+    list-style-type: none;
+    text-align: left;
+    padding-left: 131px;
+  }
+
+  .filter-btn {
+    margin-right: 11px;
+  }
+  
+}
```
Comment (lilianaweimer) [inline src/css/styles.scss:389]: Great media query!

File: src/css/_nav.scss (hunk 1 above)
Comment (H-Bo214) [reply src/css/_nav.scss:26]: I tried nesting the hover in the nav but for some reason the My Recipes and the My Pantry animations wouldn't show up when hovering over their icons.  We can look into this tomorrow. I would like to nest them if possible.

File: src/css/_variables.scss (hunk 2 above)
Comment (H-Bo214) [reply src/css/_variables.scss:4]: I read some stuff online about it, but I'm still unclear what purpose it serves. The file on my end doesn't have any weirdness going on. From what I've read I'm assuming some languages require a new line after the last entry, but I'm not exactly sure.

File: src/css/_nav.scss (hunk 1 above)
Comment (Josephhaefling) [reply src/css/_nav.scss:26]: Looks great y'all!

File: src/css/_variables.scss (hunk 2 above)
Comment (lilianaweimer) [reply src/css/_variables.scss:4]: Ok, good to know!

**Pull request merged**